guides/doc-Installing_Server_foremanctl/master.adoc

```diff
@@ -0,0 +1,12 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:installing-satellite-server-connected:
+:context: {project-context}
+:mode: connected
+:ProductName: {ProjectServer}
+:installer-foremanctl:
```

Comment:
On second thought, given that we have "foreman-installer", we might want to name it a bit differently.
```suggestion
:install-with-foremanctl:
```
Non blocking, but better for clarity of the attribute purpose.

Reply:
I meant for our clarity :) Still an optional recommendation.